src/Main.java

```diff
@@ -0,0 +1,63 @@
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        List<Integer> numbers = new ArrayList<>();
+        do {
+            System.out.println("Enter a number from 1 to 1000 ('END' to stop):");
+            String str = input.nextLine();
+            if (str.equals("END")) {
+                break;
+            }
+            int number = Integer.parseInt(str);
+            try {
+                if (number >= 1 && number <= 1000) {
+                    numbers.add(number);
+                } else {
+                    System.out.println("Invalid input");
+                }
+            } catch (NumberFormatException e) {
+                System.out.println("Insert string value");
+            }
+        } while (true);
+        Collections.sort(numbers);
+        int length = numbers.size();
+        int secoundDecile = (int) 2*(length+1)/10;
+        int thirdDecile = (int) 3*(length+1)/10;
+        int fifthDecile = (int) 5*(length+1)/10;
+        int sixthDecile = (int) 6*(length+1)/10;
+        int thirdDecileCount = 0;
+        List<Integer> third = new ArrayList<>();
+        List<Integer> sixth = new ArrayList<>();
+        int sixthDecileCount = 0;
+        int sum = 0;
+        int secoundDecileNumber = (int) numbers.get(Math.round(secoundDecile));
```

Comment:
why You round int after casting double to ind in line 26?

Reply:
Fixed